edx-solutions/edx-platform
# Enable completion and completion aggregator repos

This pulls openedx-completion-aggregator

**JIRA tickets**: OC-4474

**Discussions**: --

**Dependencies**: #1072, https://github.com/open-craft/openedx-completion-aggregator/pull/11

**Screenshots**:

**Sandbox URL**: N/A (Solutions)

**Merge deadline**: None

**Testing instructions**:

1. Fire up a ginkgo devstack with this branch.
2. Set the completion.enable_completion_tracking waffle switch.
3. Verify the site isn't broken.  Verify that creating a block completion triggers creation of a set of Aggregators.  Note: Not all blocks will create aggregators yet.  Try watching a video to the end, or manually creating a block completion.


**Author notes and concerns**:
**Reviewers**

**Settings**
